server/requirements.txt

```diff
@@ -16,7 +16,7 @@ gunicorn
 humanize
 psycopg2
 pyesbulk>=2.0.1
-PyJwt
+PyJwt[crypto]
```

Comment:
What does this really change?  Where in the code is this used?

Reply:
But we have been testing this code path already in the unit tests, right?  How has this worked up to this point?

Reply:
I used the HS256 algorithm in our auth module which uses the same key to sign a JWT and allow verification of that signature (Therefore we never needed cryptography package). Keycloak uses RS256 algorithm (an asymmetric algorithm that uses a private key to sign a JWT and a public key for the verification of that signature) Therefore the need for this change.

Reply:
As I was working on the functional test refactoring to use the Keycloak, I realized the need for this change.

Reply:
Also having this on the main already would simplify working with functional test refactoring as I wouldn't have to install cryptography package after the container is built.

Reply:
Seems like you need to modify the code so that you now use `RSA256` in the unit tests, verify doesn't work without the change, and that it does work with this change so that we know this change is sufficient.

Reply:
I have updated  `test_auth` unit tests to use the `RS256` algorithm, however, I have not updated to using `RS256` for our internal tokens. I don't think updating it is worth it right now as I will remove the current token generate mechanism and especially the way we use `secret_key` to encode the token. But `test_auth` changes are sufficient enough, _I think_, to demonstrate how `RS256` algorithm works and without the crypto library we can not use RS256.